src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
```

Comment:
since it's a list it's better to use semantic tag:
```suggestion
    <ul className="todoapp__main">
```
P.S. the same inside `TodoInfo` - use `li`